integration_tests/test_math.py

```diff
@@ -183,6 +284,8 @@ def check():
     test_expm1()
     test_log1p()
     test_trunc()
+    # test_fsum()
+    # test_prod()
```

Comment:
Should this be uncommented?

Reply:
I think we should not be adding any more functions that do not work with LPython.

Reply:
Ahh, I missed that. I'll check that again. Thanks for pointing!